dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/ClientIT.java

```diff
@@ -347,112 +347,129 @@ private void assertDepsCommandOutput() {
+        checkCommandForHelp(new String[] { WORKFLOW, SEARCH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, PUBLISH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, INFO }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, "CWL" }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, "WDL" }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, REFRESH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, LABEL }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT, CWL_2_JSON }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT, CWL_2_YAML }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT, "wdl2json" }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT, ENTRY_2_JSON }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, LAUNCH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, VERSION_TAG }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, UPDATE_WORKFLOW }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, MANUAL_PUBLISH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, "restub" }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, STAR }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, TEST_PARAMETER }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, VERIFY }, true);
+        checkCommandForHelp(new String[] { WORKFLOW }, false);
+
+        checkCommandForHelp(new String[] { PLUGIN, LIST }, true);
+        checkCommandForHelp(new String[] { PLUGIN, DOWNLOAD }, true);
+        checkCommandForHelp(new String[] { PLUGIN }, false);
+
+        checkCommandForHelp(new String[] { CHECKER }, false);
+        checkCommandForHelp(new String[] { CHECKER, DOWNLOAD }, true);
+        checkCommandForHelp(new String[] { CHECKER, LAUNCH }, true);
+        checkCommandForHelp(new String[] { CHECKER, ADD }, true);
+        checkCommandForHelp(new String[] { CHECKER, UPDATE }, true);
+        checkCommandForHelp(new String[] { CHECKER, UPDATE_VERSION }, true);
+        checkCommandForHelp(new String[] { CHECKER, TEST_PARAMETER }, true);
 
     }
 
-    private void checkCommandForHelp(String[] argv) throws IOException {
+    /**
+     * This method passes all the values in argv and --help if includeHelpFlag is true to Client.main and ensures
+     * that the output contains,
+     * 1) "Usage: dockstore", this ensures that some help page is being displayed
+     * 2) "dockstore" followed by the strings in argv, for example, if argv = {"test", "blue", "green"}, it would ensure
+     * that the output contained the string "dockstore test blue green"
+     *
+     * @param argv the arguments to pass to Client.main, do not pass "--help"
+     * @param includeHelpFlag If true, "--help" will be passed to Client.main after the arguments in argv
```

Comment:
Do you ever use this as a null? If not, would be simpler to use a `boolean`

Reply:
Good point. I have changed it to `boolean`, thanks.